packages/migrations/0004_auto_20180509_0218.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.0.2 on 2018-05-09 02:18
+
+import django.contrib.postgres.fields.jsonb
```

Comment:
can you delete all the new migrations from 004 on and regenerate it into one migration file?